src/Bitmap/ImageHeader.h

```diff
@@ -42,8 +42,9 @@ struct ImageHeader
 	void VerifyValidBitCount() const;
 	static void VerifyValidBitCount(uint16_t bitCount);
 
-	std::size_t CalculatePitch() const;
-	static std::size_t CalculatePitch(uint16_t bitCount, int32_t width);
+	// Default pitch interval is 32
```

Comment:
Suggested phrasing: "Default pitch granularity is 32 bits"
Edit: or rather, 4 bytes